### ISSUE TITLE
[Analyzer] Support splitting Wasp source code into multiple files (Wasp modules)

### ISSUE THREAD

**Issue by Martinsos** [wasp-lang/wasp#315]
This has been discussed and solution has been proposed by @craigmc08  in the https://github.com/wasp-lang/wasp/tree/master/waspc/docs/wasplang, so we should take a look there. Might be worth talking again with @craigmc08 once we start working on this.

**Comment by FrancisVarga** [comment]
2023 still open

**Comment by Martinsos** [comment]
> 2023 still open

Yup! This will actually happen quite soon probably, but not as we initially planned -> instead it will be enabled by the Wasp TS config feature that we added recently. From there, it is really a small step to have this working.

**Comment by FrancisVarga** [comment]
Looking forward to it. Would love to see close to remixjs performance its def. huge step for ai app generation and maintenance how wasp is structured.
